web/app/scripts/views/duplicates/duplicates-list-controller.js

```diff
@@ -0,0 +1,101 @@
+(function () {
+    'use strict';
+
+    /* ngInject */
+    function DuplicatesListController($scope, $rootScope, $log, $modal, $state, AuthService,
+                                      InitialState, WebConfig, Duplicates, RecordState,
+                                      RecordSchemaState ) {
+        var ctl = this;
+        ctl.currentOffset = 0;
+        ctl.numDuplicatesPerPage = WebConfig.record.limit; //Just use the records limit
+        ctl.getPreviousDuplicates = getPreviousDuplicates;
+        ctl.getNextDuplicates = getNextDuplicates;
+        ctl.showResolveModal = showResolveModal;
+        ctl.userCanWrite = false;
+
+        InitialState.ready().then(init);
+
+        function init() {
+            ctl.isInitialized = false;
+            ctl.userCanWrite = AuthService.hasWriteAccess();
+
+            RecordState.getSelected().then(function(selected) { ctl.recordType = selected; })
+                .then(loadRecordSchema)
+                .then(loadDuplicates);
+        }
+
+        function loadRecordSchema() {
+            /* jshint camelcase: false */
+            var currentSchemaId = ctl.recordType.current_schema;
+            /* jshint camelcase: true */
+
+            return RecordSchemaState.get(currentSchemaId)
+                .then(function(recordSchema) {
+                    ctl.recordSchema = recordSchema;
+                });
+        }
+
+        /*
+         * Loads a page of duplicates from the API
+         * @param {int} offset Optional offset value, relative to current offset, used
+         *                     for pulling paginated results. May be positive or negative.
+         * @return {promise} Promise to load duplicates
+         */
+        function loadDuplicates(offset) {
+            var paramsOffset;
+            if (offset) {
+                ctl.currentOffset += offset;
+                if (ctl.currentOffset) {
+                    paramsOffset = ctl.currentOffset;
+                }
+            } else {
+                ctl.currentOffset = 0;
+                paramsOffset = 0;
+            }
+
+            /* jshint camelcase: false */
+            return Duplicates.query({record_type: ctl.recordType.uuid,
```

Comment:
Yeah, I think filtering on `record_type` makes sense here; the record type represents the system's understanding of the general class of spatiotemporal occurrences that are being stored, be it Accidents, Bird Sightings, Interventions, etc. The Schema meanwhile represents the information that's being stored for each `Record`, e.g. vehicles involved in an accident. `Schema`s are versioned and can change over time, and `Record`s with different `Schema`s can all be associated with the same `RecordType`. In this case, most of the controllers in the front-end app are going to need to filter down to the `RecordType` representing Accidents, so that we don't mix Accidents together with Interventions, which are also stored as `Record` objects but of a different `RecordType` (controllers dealing with Interventions will want to limit their queries to the `RecordType` for Interventions, of course).